spec/blocksync/readme.md

```diff
@@ -0,0 +1,64 @@
+---
+order: 
+parent:
+    title: blocksync
+    order: 
+---
+
+
+In a proof of work blockchain, syncing with the chain is the same process as staying up-to-date with the consensus: download blocks, and look for the one with the most total work. In proof-of-stake, the consensus process is more complex, as it involves rounds of communication between the nodes to determine what block should be committed next. Using this process to sync up with the blockchain from scratch can take a very long time. It's much faster to just download blocks and check the merkle tree of validators than to run the real-time consensus gossip protocol.
+
+The Blocksync Reactor's high level responsibility is to enable peers who are
+far behind the current state of the consensus to quickly catch up by downloading
+many blocks (that have already been decided) in parallel, verifying their commits, and executing them against the
+ABCI application.
+
+Tendermint full nodes run the Blocksync Reactor as a service to provide blocks
```

Comment:
```suggestion
Tendermint nodes run the Blocksync Reactor as a service to provide blocks
```

Reply:
I think here you are describing the two "modes" of operation of blocksync: _server_, which ships decided blocks to whoever wants them, and _client_, which asks peers for blocks while (or when) they are needed.

I don't know whether this server/client nomenclature is good (I've used it in this [paper](https://www.inf.usi.ch/faculty/pedone/Paper/2021/srds2021a.pdf), Section V.D), but I clearly see two different roles here.